test/parallel/test-buffer.js

```diff
@@ -730,6 +730,14 @@ for (let i = 0; i < 256; i++) {
   assert.equal(hexb2[i], hexb[i]);
 }
 
+console.log('Create buffer from odd-length hex string should fail');
```

Comment:
Can you please remove the `console.log` statements, can just convert them into comments.